instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer1shim/OpenTelemetryMeterRegistryBuilder.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.micrometer1shim;
+
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.opentelemetry.api.OpenTelemetry;
+import java.util.concurrent.TimeUnit;
+
+/** A builder of {@link OpenTelemetryMeterRegistry}. */
+public final class OpenTelemetryMeterRegistryBuilder {
+
+  // Visible for testing
+  private static final String INSTRUMENTATION_NAME =
+      "io.opentelemetry.instrumentation.micrometer1shim";
```

Comment:
Should we change this back too?
```suggestion
      "io.opentelemetry.micrometer-1.5";
```

Reply:
cc @jack-berg in case this got lost in your notifications (and the long weekend)